attachment_lock/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================
+Attachment locking
+==================
+
+This module was written to allow users to lock attachments for external editing.
```

Comment:
Sorry, its not clear to me the goal for the locking.
AFAICS the doc owner can create a lock, and one hour later it will be released.
The point is to avoid concurrent edits?
What about the case you want to lock the document forever (such as when it is a final version)?

Reply:
that's in the road map, will come. Libreoffice Online will request a new lock btw, so the hour is basically the TTL. The purpose here is that we need a way to be sure nobody else edits an attachment when we edit it with external means. And I made this a module instead of hacking it into `document_wopi` itself because `document_sftp` needs just the same for proper write access.

This one will be going slow btw, because it's a pet project of mine, no day job work.